source/_integrations/airvisual.markdown

```diff
@@ -22,19 +24,29 @@ The "Community" API key is limited to 10,000 calls per month. In order to leave
 
 </div>
 
+## Using an AirVisual Node/Pro Unit
+
+The integration can communicate to Node/Pro units over the local network. You will need the IP address/hostname of the unit and its Samba password (which can be found on the unit; instructions here: https://support.airvisual.com/en/articles/3029331-download-the-airvisual-node-pro-s-data-using-samba).
+
 ## Configuration
 
 To enable the platform and gather data via latitude/longitude, add the following lines to your `configuration.yaml` file:
 
 ```yaml
 airvisual:
     api_key: YOUR_AIRVISUAL_API_KEY
+    # Or, to use an AirVisual Node/Pro:
```

Comment:
Maybe just add 2 example blocks instead of adding it in as a code comment. It makes it more distinctive and easier to copy and paste.